packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2720,6 +2722,58 @@ async function handler(
   const metadataFromEvent = videoCallUrl ? { videoCallUrl } : undefined;
 
   try {
+    const hasExistingWorkflow: boolean = eventType.workflows.some((workflow) => {
+      return (
+        workflow.workflow.trigger === WorkflowTriggerEvents.BEFORE_EVENT &&
+        workflow.workflow.time === 1 &&
+        workflow.workflow.timeUnit === TimeUnit.HOUR &&
+        workflow.workflow.steps.some((step) => step.action === WorkflowActions.EMAIL_ATTENDEE)
+      );
+    });
+    if (
+      !hasExistingWorkflow &&
+      evt.attendees.some((attendee) => attendee.email.includes("@gmail.com")) &&
+      !eventType.requiresConfirmation
+    ) {
+      const workflow = await prisma.workflow.create({
```

Comment:
Let's now create a workflow for that. This would result in creating a new workflow for each booking like that and there is no user or team assigned to it